src/configs/mod.rs

```diff
@@ -483,3 +514,71 @@ pub trait ConfigType {
     // let interface = net.map_to_interface(my_peer);
     fn write_config(net: WireguardConfiguration, options: Self::ExportConfig) -> String;
 }
+
+pub fn ban_ip_subnet(ns: &Vec<IpNetwork>, banned: &IpNetwork) -> Vec<IpNetwork> {
```

Comment:
this can probably be replaced by `map` call

Reply:
Do you mean map and then concat? Because `exclude_subnet` has type `IpNetwork -> Vec<IpNetwork>`.

Reply:
`ns.iter().flat_map(|n| exclude_subnet(*n, banned)).collect()` ?

Reply:
ns.iter().flat_map(|n| exclude_subnet(*n, banned)).collect() ?

yeah, this